west.yml

```diff
@@ -142,7 +142,7 @@ manifest:
       groups:
         - hal
     - name: hal_ambiq
-      revision: 0e15e4becd49c40cb054672d9525eccfe98b440f
+      revision: 174a4e3bbeedc7858fcaf8afaaf90b2e9d0de137
```

Comment:
Please use `pull/7/head` as a revision as described here:
https://docs.zephyrproject.org/latest/develop/modules.html#process-for-submitting-changes-to-existing-modules

Reply:
@carlescufi thanks for your comment. I updated the revision.